tico/quantization/algorithm/gptq/quantizer.py

```diff
@@ -193,7 +193,15 @@ def convert(self, model):
             )
         ):
             # 1) Identify quantizable submodules within the layer
-            full = find_layers(layer)
+            full = find_layers(
+                layer, layers=[torch.nn.Linear, torch.nn.Conv2d, torch.nn.Conv1d]
```

Comment:
If it's not a big burden, it's usually recommended that you also include test for `torch.nn.Conv1d`, too.

Reply:
Sure. I'll post it in the next PR.